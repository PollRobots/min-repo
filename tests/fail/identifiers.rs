use llvm_cov_358_min_repo::fizzbuz;

pub fn main() {
    let _ = fizzbuz! {
        1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
    };
}
