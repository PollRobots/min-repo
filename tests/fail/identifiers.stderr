error: Expecting fizz
 --> tests/fail/identifiers.rs:5:15
  |
5 |         1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
  |               ^^^^

error: Expecting buzz
 --> tests/fail/identifiers.rs:5:24
  |
5 |         1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
  |                        ^^^^

error: Expecting fizz
 --> tests/fail/identifiers.rs:5:30
  |
5 |         1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
  |                              ^^^^

error: Expecting fizz
 --> tests/fail/identifiers.rs:5:42
  |
5 |         1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
  |                                          ^^^

error: Expecting buzz
 --> tests/fail/identifiers.rs:5:47
  |
5 |         1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
  |                                               ^^^^^

error: Expecting fizz_buzz
 --> tests/fail/identifiers.rs:5:72
  |
5 |         1, 2, FIZZ, 4, BUZZ, Fizz, 7, 8, not, valid, 11, fizz, 13, 14, FizzBuzz
  |                                                                        ^^^^^^^^
